.gitignore

```diff
@@ -19,9 +19,8 @@ npm-debug.log
 # like to include it in the repository for the time being. The `dist-gem/`
 # directory though will keep being ignored for the time being.
 #
-!dist/
+dist/
 dist-gem/
```

Comment:
Remove `dist-gem/`, as it has been removed and is no longer maintained. See https://github.com/18F/us_web_design_standards_gem/issues/15#issuecomment-203564667